dist/spec.yaml

```diff
@@ -13615,6 +14273,61 @@ definitions:
     type: object
     x-okta-tags:
       - IdentityProvider
+  ProvisioningConnection:
+    properties:
+      _links:
+        additionalProperties:
+          type: object
+        readOnly: true
+        type: object
+      authScheme:
+        $ref: '#/definitions/ProvisioningConnectionAuthScheme'
+      status:
+        $ref: '#/definitions/ProvisioningConnectionStatus'
+    type: object
+    x-okta-operations:
+      - alias: getDefaultProvisioningConnectionForApplication
+        operationId: getDefaultProvisioningConnectionForApplication
+      - alias: activateDefaultProvisioningConnectionForApplication
+        operationId: activateDefaultProvisioningConnectionForApplication
+      - in: deactivateDefaultProvisioningConnectionForApplication
```

Comment:
`- alias: deactivateDefaultProvisioningConnectionForApplication`?

Reply:
fixed, thank you @VitaliiTytarenko-okta